bin/lift/Lift.cpp

```diff
@@ -432,6 +437,8 @@ int main(int argc, char *argv[]) {
 
     guide.slp_vectorize = true;
     guide.loop_vectorize = true;
+
+    CHECK(!llvm::verifyModule(dest_module, &llvm::errs()));
```

Comment:
I am not sure from the top of my head but it may be possible there is some version of that function that did `CHECK` inside itself (but maybe that was fixed some time ago) - definitely better to use existing utility though.

Reply:
[96f5b36](https://github.com/lifting-bits/remill/pull/645/commits/96f5b360926d0cbc5a803b30c16f4f62b763a148)